tests/warn/i22233.check

```diff
@@ -0,0 +1,7 @@
+-- [E194] Potential Issue Warning: tests/warn/i22233.scala:1:26 --------------------------------------------------------
```

Comment:
probably a check file is not needed, as the text is vetted by the original test.

what I would do is move tests/pos/ext-override.scala to warns.

Reply:
I removed the check file 👍 

Also I moved `tests/pos/ext-override.scala` to the warn but actually I don't understand the reasoning. The tests in  warn all give a warning right? In this test case there are no warnings. Could you elaborate?

Reply:
@RoccoMathijn they allow warn tests to have zero warnings, which is useful. I think (IMHO) tests about lack of warning should go in warn, while testing "does this compile" remain in pos. A warn test can still have `-Werror` to announce that no warns are expected (so you don't have to scan the test code).